server/game/gamesteps/challenge/fulfillmilitaryclaim.js

```diff
@@ -27,11 +36,27 @@ class FulfillMilitaryClaim extends BaseStep {
         return true;
     }
 
+    mustChooseAsClaim(card) {
+        if(_.isEmpty(this.forcedClaim)) {
+            return true;
+        }
+
+        if(this.forcedClaim.length < this.claim) {
+            return !this.forcedClaim.includes(card);
+        }
+
+        return this.forcedClaim.includes(card);
+    }
+
     fulfillClaim(p, cards) {
         if(!_.isArray(cards)) {
             cards = [cards];
         }
 
+        if(this.forcedClaim.length < this.claim) {
+            cards.push(this.forcedClaim);
```

Comment:
This pushes the `forcedClaim` array onto the `cards` array, leading `cards` to look like:

```
[ character1, [character2] ]
```

This works fine when outputting the message, but when looping through characters to kill, it sees one card and one array. I'm a bit surprised it doesn't just outright crash.

You should either loop through each card in `forcedClaim` and push them individually onto `cards`, or just set `cards` to the concatted result of it and `forcedClaim`.